local-cluster/src/HydraNode.hs

```diff
@@ -157,8 +164,15 @@ queryNode nodeId =
     (HttpExceptionRequest _ (ConnectionFailure _)) -> threadDelay 100_000 >> cont
     e -> throwIO e
 
-withHydraNode :: forall alg. DSIGNAlgorithm alg => (Int, Int, Int) -> Int -> SignKeyDSIGN alg -> [VerKeyDSIGN alg] -> (HydraClient -> IO ()) -> IO ()
-withHydraNode mockChainPorts hydraNodeId sKey vKeys action = do
+data TestLog
```

Comment:
Can we name this `LocalClusterLog` or `EndToEndLog` or anything more specific than `Test`?